bittensor release 9.6.0 (#577)
opentensor/developer-docs: docs/sdk/subtensor-api.md

| `config`          | `Config` or `None`| Pre-built Bittensor configuration object.                                                                  | `None`                         |
| `async_subtensor` | `bool`            | Whether to use the asynchronous version of the API.                                                        | `False`                        |
| `legacy_methods`  | `bool`            | If `True`, all methods from the legacy `Subtensor` class are added to this class.                           | `False`                        |

> | `fallback_endpoints` | `list[str]` or `None`| List of fallback endpoints to use if the primary network is unavailable.                                      | `None`                         |

Reviewer: ```suggestion
| `fallback_endpoints` | `list[str]` or `None`| List of fallback endpoints to use if default or provided network is not available.                                      | `None`                         |
```
